<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNormalPatient</class>
 <widget class="QDialog" name="AddNormalPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1384</width>
    <height>838</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #111111;
</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #00d2ff;</string>
   </property>
   <property name="text">
    <string>Clinic Managemnet System - Add Normal Patient</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #1c1c1c;  
border: 1px solid #333333;  
border-radius: 6px;
color: #ffffff;            
padding: 10px 15px;
font-size: 15px;
font-weight: bold;
transition: 0.45s ease;
}

QPushButton:hover {
background-color: #111111; 
border: 1px solid #00d2ff; 
}

/* 3. Optional: Pressed state for a 'clicked' feel */
QPushButton:pressed {
    background-color: #000000;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>681</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#1c1c1c&quot;;
border: none;
border-radius: 4px;
z-index: -1;
text-align: center;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>601</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Patient Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="patient_name">
       <property name="styleSheet">
        <string notr="true">/* Normal State */
QLineEdit {
    background-color: #1c1c1c;  /* Slightly lighter than #111111 background */
    border: 1px solid #333333;   /* Subtle border when not focused */
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}

/* Focus State - When the user clicks inside to type */
QLineEdit:focus {
    border: 1px solid #00d2ff;  /* Medical Cyan glow on focus */
    background-color: #222222;  /* Slightly lighter background to show activity */
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Doctor's Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Gender</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Age</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="patient_age">
       <property name="styleSheet">
        <string notr="true">/* Normal State */
QLineEdit {
    background-color: #1c1c1c;  /* Slightly lighter than #111111 background */
    border: 1px solid #333333;   /* Subtle border when not focused */
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}

/* Focus State - When the user clicks inside to type */
QLineEdit:focus {
    border: 1px solid #00d2ff;  /* Medical Cyan glow on focus */
    background-color: #222222;  /* Slightly lighter background to show activity */
}</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Disease</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="patient_disease">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="add_patient_btn">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #1c1c1c;  
border: 1px solid #333333;  
border-radius: 6px;
color: #ffffff;            
padding: 10px 15px;
font-size: 15px;
font-weight: bold;
transition: 0.45s ease;
}

QPushButton:hover {
background-color: #111111; 
border: 1px solid #00d2ff; 
}

/* 3. Optional: Pressed state for a 'clicked' feel */
QPushButton:pressed {
    background-color: #000000;
}</string>
       </property>
       <property name="text">
        <string>Add Patient</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="doctors_combo">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #1c1c1c;
    border: 1px solid #333333;
    border-radius: 4px;
    padding: 5px;
    color: #ffffff;
}

QComboBox::drop-down {
    width: 20px;
}

QComboBox::down-arrow {
    width: 12px;
    height: 12px;
}

QComboBox QAbstractItemView {
    background-color: #1c1c1c;
    border: 1px solid #333333;
    selection-background-color: #00d2ff;
    selection-color: #000000;
    color: #ffffff;
    outline: none;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="gender_picker">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: #1c1c1c;
    border: 1px solid #333333;
    border-radius: 4px;
    padding: 5px;
    color: #ffffff;
}

QComboBox::drop-down {
    width: 20px;
}

QComboBox::down-arrow {
    width: 12px;
    height: 12px;
}

QComboBox QAbstractItemView {
    background-color: #1c1c1c;
    border: 1px solid #333333;
    selection-background-color: #00d2ff;
    selection-color: #000000;
    color: #ffffff;
    outline: none;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
